### ISSUE TITLE
Accessibility: update the screen-reader-text rule to reset word-wrap

### ISSUE THREAD

**Issue by afercia** [WordPress/twentysixteen#235]
As already done in core, see https://core.trac.wordpress.org/changeset/32509 the `screen-reader-text` rule needs a `word-wrap: normal !important;` declaration. That's because elements may inherit `word-wrap: break-word;` from ancestors and that makes some screen readers read out separated words as if they were one single word.
For example, this currently happens in the Search widget which inherits `word-wrap: break-word;` from `.widget` and testing with Firefox + NVDA the hidden text gets read out as "Searchfor".

Outstanding accessibility issues related to the Search form markup should be handled in core, see `get_search_form()`.

**Comment by davidakennedy** [comment]
This is a good catch, @afercia! And probably something we should push back upstream to _s.
